sdk/automanage/ci.yml

```diff
@@ -0,0 +1,28 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
+trigger:
```

Comment:
rerun eng/scripts/Update-Mgmt-CI.ps1 to update this to the new format

Reply:
done